pubspec.yaml

```diff
@@ -8,12 +8,12 @@ documentation: https://docs.revenuecat.com/docs
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=1.12.13+hotfix.6 <2.0.0"
+  flutter: ">=1.12.13+hotfix.6"
 
 dependencies:
   flutter:
     sdk: flutter
-  collection: ^1.15.0-nullsafety.4
+  collection: ^1.15.0
```

Comment:
This is already a final release so we can update it. The warning was:

* Packages dependent on a pre-release of another package should themselves be published as a pre-release version. If this package needs collection version 1.15.0-nullsafety.4, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.